### DISCUSSION TITLE
快速入门

### DISCUSSION THREAD

**Discussion by zhongbrbr** [reata/sqllineage4j#5] [General]
请问这个有快速入门示例么，我从mvn rep 导入以下依赖，但是没有发现可用的api        

        <dependency>
            <groupId>io.github.reata</groupId>
            <artifactId>sqllineage4j</artifactId>
            <version>1.0.0</version>
            <type>pom</type>
        </dependency>

**Comment by reata** [comment]
入口类是`io.github.reata.sqllineage4j.core.LineageRunner`，可以参考测试用例中的调用：https://github.com/reata/sqllineage4j/blob/main/sqllineage4j-core/src/test/java/io/github/reata/sqllineage4j/core/Helper.java#L26
